apolloschurchapp/.gitignore

```diff
@@ -133,7 +133,6 @@ DerivedData
 *.hmap
 *.ipa
 *.xcuserstate
-project.xcworkspace
```

Comment:
can you think of why this is included in the repo now?

Reply:
I don't know, but it's part of the upgrade.